data.sql

```diff
@@ -7,7 +7,7 @@ insert into animals (name,date_of_birth,weight_kg,neutered,escape_attempts)
 ('Devimon','May 12, 2017',11,true,5)
 insert into animals (name,date_of_birth,weight_kg,neutered,escape_attempts)
  values 
-('Charmander','Feb 2, 2020',-11,false,0),
+('Charmander','Feb 2, 2020',-11,false,0) ,
```

Comment:
- [ Required ] Please consider adding `INSERT` statements in the `data.sql` file for the `vets` table as specified in the project requirements. Please see the screenshot below for more details on this.

## Project requirements
![Screenshot from 2023-01-21 02-31-57](https://user-images.githubusercontent.com/16392046/213823500-251f4b19-893b-4ecd-a1dd-346a39496dd1.png)

- [ Required ] Please consider adding `INSERT` statements in the `data.sql` file for the `specializations` table as specified in the project requirements. Please see the screenshot below for more details on this.

## Project requirements
![Screenshot from 2023-01-21 02-35-35](https://user-images.githubusercontent.com/16392046/213823789-f03bdf20-4e9b-406e-8032-ce3857c74aca.png)

- [ Required ] Please consider adding `INSERT` statements in the `data.sql` file for the `visits` table as specified in the project requirements. Please see the screenshot below for more details on this.

## Project requirements
![Screenshot from 2023-01-21 02-37-26](https://user-images.githubusercontent.com/16392046/213823934-7cfcf2b7-70ef-4cf4-91d6-1fd8e613798a.png)